### PR TITLE
Feature/moving forward added all unit tests properly

### PR DESCRIPTION
**Pull request by bmillss**
Apologies, not sure why they did not show up properly but they should show up now!

### CODE REVIEW

File: MarsRoverTest/MarsRoverTest.UnitTest/RoverTests/WhenMovingForward.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+namespace MarsRoverTest.UnitTest.RoverTests
+{
+    [TestClass]
+    public class WhenMovingForward
+    {
+        [TestMethod]
+        public void AndFacingNorthThenYIncrementsByOne()
+        {
+            //arrange 
+            var rover = new Rover();
```
Comment (cameronpresley) [inline MarsRoverTest/MarsRoverTest.UnitTest/RoverTests/WhenMovingForward.cs]: Total nitpick, but we can combine these two statements to a single line as 
`var rover = new Rover{Direction=Direction.North}`

File: MarsRoverTest/MarsRoverTest/Rover.cs [hunk 2]
```diff
@@ -20,10 +20,27 @@ public Rover()
...
+        {
+            if (Direction == Direction.North)
+            {
+                Y = Y + 1;
+            }
+            else if (Direction == Direction.South)
+            {
+                Y = Y - 1;
+            }
+           else if (Direction == Direction.East)
```
Comment (cameronpresley) [inline MarsRoverTest/MarsRoverTest/Rover.cs]: Total nitpick here as well, can we get this `else if` lined up correctly? You can Visual Studio do this automatically for you by using ctrl+k followed by ctrl+d (or by going to the menu bar -> Edit -> Advanced -> Format Document)

**Review by cameronpresley — COMMENTED** [review]
Overall, the changes look solid. Good naming, the code is of adequate quality and I can follow what your intent was. There are two very small things to change, but otherwise, this looks good. Excellent work!

File: MarsRoverTest/MarsRoverTest.UnitTest/RoverTests/WhenMovingForward.cs (hunk 1 above)
Comment (bmillss) [reply MarsRoverTest/MarsRoverTest.UnitTest/RoverTests/WhenMovingForward.cs]: Hey Cameron this change has been done for each direction in order to make things look cleaner thanks for the heads up on the cleaner syntax!

**Comment by cameronpresley** [comment]
Changes look great, go ahead and complete!

**Pull request merged**